packages/valory/agents/registration_start_up/aea-config.yaml

```diff
@@ -60,26 +66,40 @@ public_id: valory/abci:0.1.0
 type: connection
 config:
   target_skill_id: valory/registration_abci:0.1.0
+  host: ${ABCI_HOST:str:localhost}
+  port: ${ABCI_PORT:int:26658}
+  use_tendermint: ${ABCI_USE_TENDERMINT:bool:false}
 ---
 public_id: valory/ledger:0.1.0
 type: connection
 config:
   ledger_apis:
     ethereum:
-      address: ${LEDGER_ADDRESS:str:http://127.0.0.1:8545}
+      address: ${LEDGER_ADDRESS:str:https://staging.chain.autonolas.tech}
       chain_id: ${LEDGER_CHAIN_ID:int:31337}
 ---
+public_id: valory/registration_abci:0.1.0
+type: skill
+models:
+  benchmark_tool:
+    args:
+      log_dir: ${SKILL_ORACLE_ABCI_MODELS_BENCHMARK_TOOL_ARGS_LOG_DIR:str:/benchmarks}
+  params:
+    args:
+      tendermint_url: ${TENDERMINT_URL:str:http://localhost:26657}
+      tendermint_com_url: ${TENDERMINT_COM_URL:str:http://localhost:8080}
+---
 public_id: valory/p2p_libp2p_client:0.1.0
 type: connection
 config:
   nodes:
-  - uri: ${P2P_URI:str:staging.acn.autonolas.tech:9005}
-    public_key: ${P2P_PUBLIC_KEY:str:0250827315351f6f4d3777b77d4bfd67280156240313bf221750b36404344bb35d}
+  - uri: ${P2P_URI:str:staging.acn.autonolas.tech:9006}
+    public_key: ${P2P_PUBLIC_KEY:str:02fc5fe35c21a1f8ba2de52fd38ef95379be19d5390f1275b825c343167590d3b7}
```

Comment:
`9005` was not working. Any idea why @Karrenbelt @8ball030 ?

Reply:
so now this is working again? I suggest to try adding both genesis nodes to see if the agents can communicate even when they connect to different genesis nodes